app/src/androidTest/java/org/mozilla/tv/firefox/ui/BasicNavigationTest.kt

```diff
@@ -134,4 +135,14 @@ class BasicNavigationTest {
             assertActivityFinishing(activityTestRule)
         }
     }
+    @Test
+    fun screenNavigationFocusTest() {
+        // Navigate to settings via keypresses, to maintain focus.
+        navigationOverlay {
+        }.enterUrlAndEnterToBrowser(Uri.parse("firefox:about")) {
+        }.openOverlay {
+        }.linearNavigateToSettingsAndOpen {
```

Comment:
Nit: I think `// Navigate to settings via keypresses, to maintain focus.` might make more sense just above line 144.